docs/src/pages/[platform]/studio/figma.page.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import styles from 'src/styles/studio/figma.module.scss';
+import { useNavigationContext } from 'src/hooks/useNavigationContext';
+import { Button, Image, Text, TextField } from '@aws-amplify/ui-react';
+import { Features } from '@/components/Studio/Features';
+
+const FigmaToCode = () => {
+  const { setAlwaysCollapsible } = useNavigationContext();
+  setAlwaysCollapsible(true);
```

Comment:
Not sure what `setAlwaysCollapsible` is doing, but should it be running on every render?

Reply:
But should it be called on every render? And when (if ever) does `alwaysCollapsible` get set back to `false`?